mlflow/R/mlflow/.run-tests.R

```diff
@@ -8,8 +8,16 @@ use_condaenv(mlflow:::mlflow_conda_env_name())
 
 devtools::check_built(
     path = package,
+    cran = TRUE,
     remote = should_enable_cran_incoming_checks(),
     error_on = "note",
     args = "--no-tests"
 )
+# This runs checks that are disabled when `cran` is TRUE (e.g. unused import check).
+devtools::check_built(
+    path = package,
+    cran = FALSE,
+    error_on = "note",
+    args = "--no-tests"
+)
```

Comment:
This took 35.3s: https://github.com/mlflow/mlflow/runs/4716826204?check_suite_focus=true#step:13:197